src/helpers/SUE/sueParser.tsx

```diff
@@ -30,15 +34,23 @@ const parseSueRequests = (data) => {
       email: item?.email,
       firstName: item?.first_name,
       lastName: item?.last_name,
-      position: { longitude: item?.long, latitude: item?.lat },
+      params: {
+        details: { ...item },
+        query: QUERY_TYPES.SUE.REQUESTS_WITH_SERVICE_REQUEST_ID,
+        queryVariables: { id: item.service_request_id },
+        rootRouteName: ROOT_ROUTE_NAMES.SUE,
+        title: texts.detailTitles.pointOfInterest
```

Comment:
shouldn't this be the title of the item?

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/960/commits/df11b26e53a0ba1d4d9529a1c5c15e73e36aadb8